srcs/server/Response.cpp

```diff
@@ -61,11 +53,9 @@ void    Response::setDateHeader(void) {
 }
```

Comment:
in line 44 - `nullptr` is a c++11 keyword apparently
![image](https://user-images.githubusercontent.com/36924670/212545159-bdcbefb6-68a8-410b-b7b1-ca03b5d8be85.png)